gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 kotlinVersion = "1.8.10"
-androidGradleVersion = "7.4.2"
-androidSdkCommonVersion = "30.4.2"
+androidGradleVersion = "8.0.1"
```

Comment:
we should support AGP 8 and 7 at same time. to test it we can use different AGP in samples - for example in `resource-gallery` use old 7.4.2 but in `compose-resource-gallery` use new 8.0.1.
current implementation will works with old AGP too?